packages/cli/dependencies/run/index.js

```diff
@@ -4,5 +4,9 @@ export default (({
 }) => () => {
   console.log(logo())
   console.log(`BMO CLI v ${version}`)
-  program.parseAsync(process.argv)
+  if (program.parseAsync) {
```

Comment:
What is this checking for?
```sugestion
return program.parseAsync
  ? program.parseAsync(process.argv)
  : program.parse(process.argv)
```

Reply:
newer versions of commander have a parse async function that is preferable over the parse so this uses that if available.

Reply:
I don't really see a benefit of the ternary vs the if else personally.